novawallet/Modules/Vote/Governance/ReferendumVoteSetup/ReferendumVoteSetupPresenter.swift

```diff
@@ -308,15 +313,24 @@ final class ReferendumVoteSetupPresenter {
             factory: dataValidatingFactory,
             params: params,
             selectedLocale: selectedLocale,
+            convictionUpdateClosure: { [weak self] in
```

Comment:
6 parameters to much. I would make a separate model to provide handlers: fee and conviction update